src/index.js

```diff
@@ -134,24 +141,34 @@ export default function(options = {}) {
 
               if (getExport) {
                 codeExportDefault = getExport(result.opts.from)
-                Object.keys(codeExportDefault).forEach(k => {
-                  const camelCasedKey = dashesCamelCase(k)
-                  if (reserved.check(camelCasedKey)) {
-                    console.warn(
-                      chalk.yellow('You are using a reserved keyword'),
-                      chalk.cyan(camelCasedKey),
-                      chalk.yellow(
-                        "as className so it's not available in named exports"
+
+                if (escapeDashes) {
+                  const escapedExportDefault = {}
+                  Object.keys(codeExportDefault).forEach(k => {
+                    const camelCasedKey = dashesCamelCase(k)
+                    escapedExportDefault[camelCasedKey] = codeExportDefault[k]
+                  })
+                  codeExportDefault = escapedExportDefault
+                }
+                if (exportNamed) {
```

Comment:
If `getExport` is not defined, nothing is exported, so it's not quite the same.

My reason for creating this PR is to not have to deal with any troubles arising from the limitations on the names of named exports.

Add to that, in my opinion it is quite confusing that the class names in the default export would be different from the named exports. What is the use for that?

If what you are after is having the `namedExport` toggle both the named exports and the escaping, then that's fine with me.